### PR TITLE
(#231) Add license upgrade documentation

### PR DESCRIPTION
**Pull request by steviecoaster**
We currently don't have documentation on upgrading a license.
This PR adds documentation with the steps needed to complete this task.

### CODE REVIEW

**Comment by gep13** [comment]
@steviecoaster where did you see this new page appearing? Should it be linked to from another page, or as a new entry in the left hand navigation?

File: QuickDeployment/V2/QuickDeploymentUpgradeLicense.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Upgrade License on Quick Deployment Environment
```
Comment (vexx32) [inline QuickDeployment/V2/QuickDeploymentUpgradeLicense.md:1]:
```suggestion
# Upgrading your license in the Quick Deployment Environment
```

File: QuickDeployment/V2/QuickDeploymentUpgradeLicense.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+You may receive a trial extension license, or a new license when you renew your currently purchased license. Upon receiving that new file, you will need to upgrade the `chocolatey-license` package in your Nexus repository (or repository server if you've moved to your own separate from QDE).
+
+On the QDE server in the `C:\choco-setup\files` folder you will find a `CreateLicensePackage.ps1` script. This script will have been updated when you provisioned the QDE server with the proper hostname for your environment. This script will package the new license file on the server and push it to your repository.
+
+# Upgrade Process
+
+Steps:
```
Comment (vexx32) [inline QuickDeployment/V2/QuickDeploymentUpgradeLicense.md:9]:
```suggestion
```

File: QuickDeployment/V2/QuickDeploymentUpgradeLicense.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+
```
Comment (vexx32) [inline QuickDeployment/V2/QuickDeploymentUpgradeLicense.md:10]:
```suggestion

```

File: QuickDeployment/V2/QuickDeploymentUpgradeLicense.md [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+1. Apply new license file named `chocolatey.license.xml` to the `C:\ProgramData\chocolatey\license` folder ensuring that it is the _only_ file in the directory.
```
Comment (vexx32) [inline QuickDeployment/V2/QuickDeploymentUpgradeLicense.md:11]:
```suggestion
1. Copy the new license file named `chocolatey.license.xml` to the `C:\ProgramData\chocolatey\license` folder ensuring that it is the _only_ file in the directory.
```

File: QuickDeployment/V2/QuickDeploymentUpgradeLicense.md [hunk 5]
```diff
@@ -0,0 +1,13 @@
...
+2. Run `& C:\choco-setup\files\CreateLicensePackage.ps1 -NexusApiKey 'PutYourApiKeyHere'`
+3. On _exsting provisioned_ client workstations run `choco upgrade chocolatey-license`. Newly provisioned clients from this point forward will apply the correct updated license file.
```
Comment (vexx32) [inline QuickDeployment/V2/QuickDeploymentUpgradeLicense.md:13]:
```suggestion
3. On any _already provisioned_ client workstations run `choco upgrade chocolatey-license` to have them retrieve and apply the new license file automatically. Newly provisioned clients from this point forward will use the newly updated license file.
```

**Comment by ferventcoder** [comment]
this looks good based on suggestions and comments from others

**Comment by steviecoaster** [comment]
Requested changes have been made.

**Comment by gep13** [comment]
@steviecoaster did you see the question that I asked? If this page needs to be added to the navigation menu, this will take some extra work in the website repository.

**Comment by steviecoaster** [comment]
Under the QDE documentation. It's applicable (more or less) to both the v1 and v2 appliance so under there _somewhere_ would be good.

**Review by gep13 — APPROVED** [review]
LGTM!

**Comment by gep13** [comment]
@steviecoaster thanks for getting this added!

**Pull request merged**